helper-npm-version-and-publish-public

```diff
@@ -10,5 +10,10 @@ set -eu -o pipefail
 
 # HELPER COMMANDS
 
+PRERELEASE_PATTERN='v[0-9]+\.[0-9]+\.[0-9]+-[a-z]+\.[0-9]+'
```

Comment:
is this the pattern everyone has been using? it's more restrictive than semver's definition of pre-release

Reply:
It's taken from another project... it does look quite strict! Perhaps checking only for the existence of an identifier will suffice, `"-[a-z]+"` ?